### ISSUE TITLE
QualityDifferrence

### ISSUE THREAD

**Issue by thisisawesome1994** [TheDeezLoader/pyDeezloader#2]
Hi,

Could you fix the ffmpeg command, OR whatever command you used. The quality of the downloaded Flac is not the same as the one from the app on my android, OR hifi stereo for that matter.

To fix, use no compression(1411kbps)

**Comment by thisisawesome1994** [comment]
No, nevermind... I think windows is reporting wrong bitrate...

**Comment by thisisawesome1994** [comment]
No, it is still compressed i think...

**Comment by BlackStoneReborn** [comment]
Did you really specify the quality?

**Comment by thisisawesome1994** [comment]
yeah, but FLAC is not lossless. People say it is, but it is not. 
Probably it is the issue of Deezer.

Do you know if Deezer stores the uncompressed file somewhere accessible?
